LoopKitUI/View Controllers/AuthenticationViewController.swift

```diff
@@ -62,14 +62,7 @@ public final class AuthenticationViewController<T: ServiceAuthenticationUI>: UIT
     }
 
```

Comment:
Return all credentials --whether authorized or not. This will allow us to show a field with ***** for the secret. Before this we would just show the username. This seemed helpful to remind users whether the service was setup with or without a secret, since it is an optional field. I'm thinking of the case specifically where a user decided to add or remove a secret field requirement on their Nightscout instance, then coming back into the app and not being sure why the Nightscout Api Client is failing.